### PR TITLE
add example of get() function

### PR DESCRIPTION
**Pull request by FredSoftwares**
very usefull to extract the value from the Nullable wrapper.

### CODE REVIEW

File: docs/src/man/getting_started.md [hunk 1]
```diff
@@ -23,6 +23,31 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function is very usefull to extract the value from the Nullable wrapper.
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: I would just say "can be used to". `Nullable` should be in backticks. Also mention "when it is not null".

File: docs/src/man/getting_started.md [hunk 2]
```diff
@@ -23,6 +23,31 @@ Nullable()
...
+DateTime
+
+julia> c = get(a)
+"14:00:00"
+
+julia> typeof(c)
+String
+
+```
+
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: One line break is enough.

File: docs/src/man/getting_started.md [hunk 3]
```diff
@@ -23,6 +23,31 @@ Nullable()
...
+example :
+```julia
+julia> a = Nullable{String}("14:00:00")
+Nullable{String}("14:00:00")
+
+julia> typeof(a)
+Nullable{String}
+
+julia> b = Dates.DateTime(get(a), "HH:MM:SS")
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: No need to introduce date/time functionalities here. Just show that the `String` value can be extracted.

**Review by nalimilan — CHANGES_REQUESTED** [review]
Thanks. Could you also add a reference to the manual? http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values

File: docs/src/man/getting_started.md [hunk 4]
```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
+
+example :
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: Missing initial upper case. Also, better not create a new paragraph just for this word.

File: docs/src/man/getting_started.md [hunk 5]
```diff
@@ -23,6 +23,21 @@ Nullable()
...
+```julia
+julia> a = Nullable{String}("14:00:00")
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: `{String}` isn't needed here.

File: docs/src/man/getting_started.md [hunk 6]
```diff
@@ -23,6 +23,21 @@ Nullable()
...
+Nullable{String}("14:00:00")
+
+julia> c = get(a)
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: Call this `b`?

File: docs/src/man/getting_started.md [hunk 7]
```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: Ah, and missing backticks around `get`. Also I think in general the parentheses are omitted.

File: docs/src/man/getting_started.md (hunk 6 above)
Comment (FredSoftwares) [reply docs/src/man/getting_started.md]: ok :) 
I did not understand what you mean by "add a reference to the manual?" do you mean I should insert a link (just paste http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values) or do I have to add the example of the get() function in the stable manual ? Sorry, I am very new to github.

Comment (nalimilan) [reply docs/src/man/getting_started.md]: I meant adding a sentence pointing to the Julia manual for more information (the URL shouldn't appear, it should just be a link).

**Comment by nalimilan** [comment]
I think you missed a few other comments I've made.

File: docs/src/man/getting_started.md [hunk 8]
```diff
@@ -23,9 +23,8 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
-The get() function can be used to extract the value from the `[Nullable](http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values)` wrapper when it is not null.
+The `get()` function can be used to extract the value from the `[Nullable](http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values)` wrapper when it is not null. For example:
 
-example :
 ```julia
 julia> a = Nullable{String}("14:00:00")
```
Comment (nalimilan) [inline docs/src/man/getting_started.md]: `{String}` is still not needed here.

**Review by ararslan — APPROVED** [review]

**Review by nalimilan — APPROVED** [review]
Thanks @FredSoftwares!

**Pull request merged**